components/info.jsx

```diff
@@ -0,0 +1,35 @@
+function Info() {
+  return (
+    <>
+      <div className="nav-container">
+      <nav >
+        <ul >
+          <li className="gpt"><a href="#">GPT-3</a></li>
+          <li><a href="#">HOME</a>  </li>
+          <li> <a href="#">what is GPT?</a></li>
+          <li> <a href="#">open Al</a> </li>
+          <li> <a href="#">case studies</a> </li>
+          <li> <a href="#">Library</a></li>
+        </ul>
+      </nav>
+      <button className="sign"> sign in</button>
+      <button>login </button>
+    </div>
+    <img src="images/ai.png"></img>
+    <div  >
+      <h2 >Let's Build something <br/> amazing with GPT-3 openAl </h2>
+      <p>Yet bad any for travelling assistance indulgence unpleassing not throughts all exercise blessing way <br/>
+       everything joy altereation boisterous the attachment  party<br/> we years to order allow asked of. </p>
+      <button className="email">enter your email </button>
```

Comment:
![Capture](https://github.com/AASB7/Training_Project/assets/96662087/46318c25-2d89-4ed2-8ee0-701ae0f5c38e)
Please fix the design